src/gatsby/utils/models/index.js

```diff
@@ -0,0 +1,36 @@
+/** Model API runner
+  (
+    models: Array<Model>
+      The array of models to run the action on.
+    name: string
+      If any model has a function under under this key, that function will be run.
+    api: Object<GatsbyAPI>
+      The api object provided by Gatsby in whichever lifecycle method or Model this
+      function is being called in.
+    modelOptions
+      This fourth positional parameter is meant to pass data through to the Models.
+      If this isn't provided, it defaults to an object including all the models
+      currently being run as an injected dependency: this allows for sharing
+      Gatsby build code via models that call custom hooks on other models.
+  )
+
+  Standard usage:
+    In a Gatsby API, this mimics the original Gatsby call on all Models:
+
+    exports.onCreateNode = (api) =>
+      asyncCallOnAll(models, "onCreateNode", api)
```

Comment:
update the name, also probably update it in the code runOnModels -> callOnModels ?

Reply:
may be I'm missing some semantics behind the `runOnModels` name though ...

Reply:
You aren't missing anything, this is just me be scatterbrained with a default import that got its name changed a few times. I've got a fix coming in hot.